e2e/testcafe-devextreme/tests/scheduler/grouping/overflow.ts

```diff
@@ -0,0 +1,80 @@
+import { createScreenshotsComparer } from 'devextreme-screenshot-comparer';
+import Scheduler from 'devextreme-testcafe-models/scheduler';
+import { changeTheme } from '../../../helpers/changeTheme';
+import { createWidget } from '../../../helpers/createWidget';
+import url from '../../../helpers/getPageUrl';
+
+fixture.disablePageReloads`Scheduler: SmoothCellLines`
+  .page(url(__dirname, '../../container.html'));
+
+test('Long appointments should not overflow group view', async (t) => {
+  const scheduler = new Scheduler('#container');
+  const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+  await t
+    .expect(await takeScreenshot('group-overflow', scheduler.element))
```

Comment:
It's better to write the file extension
```suggestion
    .expect(await takeScreenshot('group-overflow.png', scheduler.element))
```